autoendpoint/src/server/extractors/subscription.rs

```diff
@@ -58,8 +58,8 @@ impl FromRequest for Subscription {
             }
 
             // Load and validate user data
-            let uaid = Uuid::from_slice(&token[..16])?;
-            let channel_id = Uuid::from_slice(&token[16..32])?;
+            let uaid = Uuid::from_slice(&token[..16]).unwrap();
```

Comment:
I'll bite, why `unwrap()` rather than handle the Uuid error?

Reply:
An error will only be returned if the slice length is not 16:
https://docs.rs/uuid/0.8.1/uuid/struct.Uuid.html#method.from_slice

I can add a clarifying comment.